okhttp/src/main/java/okhttp3/internal/ws/WebSocketReader.java

```diff
@@ -46,8 +46,8 @@
 
 /**
  * An <a href="http://tools.ietf.org/html/rfc6455">RFC 6455</a>-compatible WebSocket frame reader.
- * <p>
- * This class is not thread safe.
+ *
+ * <p>This class is not thread safe.
```

Comment:
๏_๏

Reply:
Hmm. Apparently I’m inclined to do it the Google way, which is different than the way specified by Javadoc.

https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html

Sigh.

Reply:
Yeah we talked about it once. I think we determined it was broken in an older Javadoc version, but I don't really remember. Also I don't actually care either way which is used. I'd prefer to be consistent and it looks like this style is used more throughout the library.